Assets/MixedRealityToolkit.Examples/Demos/Gltf/Scripts/TestGltfLoading.cs

```diff
@@ -19,13 +20,19 @@ namespace Microsoft.MixedReality.Toolkit.Examples.Demos.Gltf
     public class TestGltfLoading : MonoBehaviour
     {
         [SerializeField]
+        [FormerlySerializedAs("uri")]
         [Tooltip("The relative asset path to the glTF asset in the Streaming Assets folder.")]
-        private string uri = "\\GltfModels\\Lantern\\glTF\\Lantern.gltf";
+        private string relativePath = "\\GltfModels\\Lantern\\glTF\\Lantern.gltf";
 
-        public string Uri
-        {
-            get => uri;
-        }
+        /// <summary>
+        /// The relative asset path to the glTF asset in the Streaming Assets folder.
+        /// </summary>
+        public string RelativePath => relativePath.NormalizeSeparators();
+
+        /// <summary>
+        /// Combines Streaming Assets folder path with RelativePath
+        /// </summary>
+        public string AbsolutePath => Path.GetFullPath(Application.streamingAssetsPath) + RelativePath;
```

Comment:
Should this do a `Path.Combine` too, instead of a `+`? In case the relative path was entered without a slash. Unless `Application.streamingAssetsPath` guarantees one?